packages/rum/test/specs/apm-base.spec.js

```diff
@@ -308,6 +308,21 @@ describe('ApmBase', function() {
     expect(tr.spans[0].name).toBe('GET /')
   })
 
+  it('should allow setting labels before calling init', () => {
+    const labels = {
+      foo: '1',
+      bar: 2
+    }
+    apmBase.addLabels(labels)
+    apmBase.init({
+      serviceName,
+      serverUrl,
+      disableInstrumentations: [PAGE_LOAD]
+    })
+    const configService = serviceFactory.getService('ConfigService')
+    expect(configService.get('context.tags')).toEqual(labels)
+  })
```

Comment:
I think having this test here is fine but since this change is in config service, the test should be in it's spec file. there can be two test if you think that's justified.

Reply:
There is already a test in config service, so I added this test to ensure setting labels before init.